testing/trino-product-tests/README.md

```diff
@@ -144,11 +144,11 @@ This enables users to test features in a cross functional way. To run a
 particular group, use the `-g` argument as shown:
 
 ```
-# Run all tests in the string_functions and create_table groups
+# Run all tests in the functions and create_table groups
 testing/bin/ptl test run \
             --environment <environment> \
             [--config <environment config>] \
-            -- -g string_functions,create_tables
+            -- -g functions,create_tables
 ```
```

Comment:
wrong commit

Reply:
leftover -- i was trying to cleanup groups in convention based tests, but decided it's better not to do this in this PR. will revert.